python/ray/rllib/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 
 def _register_all():
-    for key in ["PPO", "ES", "DQN", "APEX", "A3C", "BC", "PG", "__fake",
+    for key in ["DDPG", "APEX_DDPG", "PPO", "ES", "DQN", "APEX", "A3C", "BC", "PG", "__fake",
```

Comment:
Let's call this `DDPG2` and `APEX_DDPG2` for now, since there is a conflicting PR that was merged earlier. We will resolve the differences later to combine the implementations.

The package directory should also be moved to `rllib/ddpg2`.

Reply:
Done.